packages/stark-ui/src/modules/session-ui/session-ui.module.ts

```diff
@@ -40,31 +53,25 @@ export class StarkSessionUiModule {
 	 * @link https://angular.io/guide/singleton-services#forroot
 	 * @returns a module with providers
 	 */
-	public static forRoot(): ModuleWithProviders {
+	public static forRoot(starkSessionUiConfig?: StarkSessionUiConfig): ModuleWithProviders {
 		return {
-			ngModule: StarkSessionUiModule
+			ngModule: StarkSessionUiModule,
+			providers: [starkSessionUiConfig ? { provide: STARK_SESSION_UI_CONFIG, useValue: starkSessionUiConfig } : []]
 		};
 	}
 
 	/**
 	 * Prevents this module from being re-imported
 	 * @link https://angular.io/guide/singleton-services#prevent-reimport-of-the-coremodule
 	 * @param parentModule - The parent module
-	 * @param translateService - the translation service of the application
 	 */
 	public constructor(
 		@Optional()
 		@SkipSelf()
-		parentModule: StarkSessionUiModule,
-		translateService: TranslateService
+		parentModule: StarkSessionUiModule
 	) {
 		if (parentModule) {
 			throw new Error("StarkSessionUiModule is already loaded. Import it in the AppModule only");
 		}
-
-		const english: StarkLocale = { languageCode: "en", translations: translationsEn };
-		const french: StarkLocale = { languageCode: "fr", translations: translationsFr };
-		const dutch: StarkLocale = { languageCode: "nl", translations: translationsNl };
-		mergeUiTranslations(translateService, english, french, dutch);
```

Comment:
I think this is why the translations are not working anymore in the Session UI module :p    Maybe a rebase problem?